test/mobx.js

```diff
@@ -191,21 +209,30 @@ describe('usage with mobx should generally work', () => {
       },
     })
     expect(service).to.have.callCount(2)
-    expect(reactor).to.have.callCount(5)
+    expect(reactor).to.have.callCount(26)
     expect(Tree.getNode(['root', 'newFilterWithValueForRemoveTest'])).to.exist
 
     await Tree.remove(['root', 'newFilterWithValueForRemoveTest'])
     expect(Tree.getNode(['root', 'newFilterWithValueForRemoveTest'])).to.not
       .exist
     expect(service).to.have.callCount(3)
-    expect(reactor).to.have.callCount(7)
+    expect(reactor).to.have.callCount(42)
```

Comment:
O_o

Reply:
Since the properties are extended from the beginning, and the reaction is triggered with any property change (since we're doing toJS to the whole object), this number is correct.

Reply:
since these numbers basically mean nothing anymore, let's find a way for them not be in the tests explicitly (e.g. just assert call count is going up regardless of how many times)